spec/integration_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe 'integration test' do
+  xit 'should not raise error' do
```

Comment:
Could you have written about future features/testing and any unused tests in the readme?

Reply:
For an integration or feature test, you could execute the methods to replicate a full user journey - placing an order and confirming the order total.